vignettes/vignette.Rnw

```diff
@@ -518,6 +531,16 @@ pathway.visualization(aCML,
 \caption{\texttt{oncoprint} output of a custom pair of pathways, with events shown in top and hidden in bottom.}
 \end{figure*}
 
+And you can view and edit these pathways at WikiPathways using these commands to open tabs in your default browser.
+
+<<wikipathways>>=
+browseURL(getPathwayInfo(my.wpids[1])[2])
```

Comment:
It is not a good idea to put something "web" in the vignette. This could be a problem in case of trouble with the connection or any other issue on this level. In general could be better to mock this type of code and insert the output in a separate block

Reply:
I disagree. The vast majority of users will be online when working with TRONCO and going through vignettes. And even if offline, this code simply doesn't open a tab. No harm to the rest of the vignette. In fact, users can just skip them since they are clearly annotated as having to do with opening pages in a browser. And the value of pointing out that these are real pathways that can be usef for many application areas of TRONCO is worthwhile.